diff --git a/dist/connect.js b/dist/connect.js
index dea6775064afe229884c1c6d1e07dba5577cbd96..034b6b205520edeb9fe705d6ebe62e803516455c 100644
--- a/dist/connect.js
+++ b/dist/connect.js
@@ -1,6 +1,6 @@
-import { createClient } from '@libsql/client';
+import { createClient } from '@libsql/client/web';
 import { pushDevSchema } from '@payloadcms/drizzle';
-import { drizzle } from 'drizzle-orm/libsql';
+import { drizzle } from 'drizzle-orm/libsql/web';
 export const connect = async function connect(options = {
     hotReload: false
 }) {
